qiskit_qec/operators/base_xp_pauli.py

```diff
@@ -269,8 +328,70 @@ def _compose(
         front: bool = False,
         inplace: bool = False,
     ) -> "BaseXPPauli":
-        """_summary_"""
-        pass
+        """Returns the composition of two BaseXPPauli objects.
+
+        Args:
+            a : BaseXPPauli object
+            b : BaseXPPauli object
+            qargs (Optional[list], optional): _description_. Defaults to None.
+            front (bool, optional): _description_. Defaults to False.
+            inplace (bool, optional): _description_. Defaults to False.
+
+        Returns:
+            BaseXPPauli: _description_
+        """
+
+        assert a.precision == b.precision, QiskitError(
```

Comment:
This should not be in a "_"-method but a higher method `compose` which probably belongs in XPPauli and XPPauliList (different for each). Assertions have the nice property that they have zero impact with DEBUG is set correctly but some people feel that they should not be used for format requirements and are more to detect algorithm and internal faults.

Reply:
See previous comment. 
Replaced `assert` with `if...raise`.